Example/Images.xcassets/image_placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "2.png",
```

Comment:
This is a placeholder image that will be shown before the icon has finished downloading?

Reply:
Yes, Looks like the image name inside the json was wrong. I have updated in the new commit now.